### PR TITLE
Added new package r-fasterize at verion 1.0.3

### PR DESCRIPTION
**Pull request by alexander-baumgarten**

### CODE REVIEW

File: var/spack/repos/builtin/packages/r-fasterize/package.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    homepage = "https://cran.r-project.org/package=fasterize"
+    url      = "https://cran.r-project.org/src/contrib/fasterize_1.0.3.tar.gz"
+
+    version('1.0.3', sha256='62b459625e9bdb00251ec5f6cb873e0c59713f3e86dc1e2c8332adc0cea17f81')
+
+    depends_on('r@3.3.0:')
+    depends_on('r-rcpp')
+    depends_on('r-raster')
+    depends_on('r-sp')
+    depends_on('r-rcpparmadillo')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/r-fasterize/package.py]: Need to add `type=('build', 'run')` to all of these

File: var/spack/repos/builtin/packages/r-fasterize/package.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+
+from spack import *
+
+
+class RFasterize(RPackage):
+    """Provides drop-in replacement for rasterize() from the package raster."""
+
+    homepage = "https://cran.r-project.org/package=fasterize"
+    url      = "https://cran.r-project.org/src/contrib/fasterize_1.0.3.tar.gz"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/r-fasterize/package.py:13]: Needs a `list_url`: https://spack.readthedocs.io/en/latest/build_systems/rpackage.html#list-url

Also, you should use `cloud` over `cran` for mirrors

**Review by adamjstewart — CHANGES_REQUESTED** [review]

File: var/spack/repos/builtin/packages/r-fasterize/package.py (hunk 2 above)
Comment (adamjstewart) [reply var/spack/repos/builtin/packages/r-fasterize/package.py:13]: Can you use cloud for the homepage and url too?

**Comment by adamjstewart** [comment]
@glennpj

File: var/spack/repos/builtin/packages/r-fasterize/package.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class RFasterize(RPackage):
+    """Provides drop-in replacement for rasterize() from the package raster."""
+
+    homepage = "https://cran.r-project.org/package=fasterize"
```
Comment (glennpj) [inline var/spack/repos/builtin/packages/r-fasterize/package.py:12]: This should reflect the URL, if there is one, in the DESCRIPTION file. This should also be reflected on the CRAN packages page. The URL, and homepage for spack, should be:

https://github.com/ecohealthalliance/fasterize

**Comment by alalazo** [comment]
@alexander-baumgarten What is the status of this PR?

**Comment by alalazo** [comment]
Closing since there was no reply in  a few months. Feel free to reopen in case you want to continue working on the PR.

**Pull request closed without merging**